src/models/sections/__tests__/identificationDateOfBirth.test.js

```diff
@@ -70,4 +70,22 @@ describe('The identification date of birth section', () => {
 
     expect(validateModel(testData, identifcationDateOfBirth)).toBe(true)
   })
+
+  it('allows an "invalid" birthdate if confirmed', () => {
```

Comment:
name of the test needs to be updated because birthdays over 100 are not allowed.

Reply:
currently the model allows anything to pass if the new field confirmed is true, i think we can add the restrictions in EN-3633 since that ticket is dealing with the age limit 16-100 constraints

Reply:
Not quite sure I understand. EN-3633 does not mention of anything about age minimums or limits. It's specifically referencing a use case where there is no error then the year is `1000`.

This PR was created to fix a defect that was introduced by #1840.

Reply:
Ya, EN-3633 was 1000 but that's where I discovered the age limit issue, which is related to entering a year 1000, because entering a 1000 year means you're over the age of 100, which should error, but wasn't. Perhaps we should create a separate issue?